### PR TITLE
Feature/socioeconomic indicators population

### PR DESCRIPTION
**Pull request by Bluesmile82**
Socioeconomic page:

- Add Population, economy an energy
- Update section title
- Add switch component that styles the cw-component
- Add button to power explorer, still missing link.

We might want to double check the indicators shown as they are handpicked from the available ones. Not sure if these are final

Extra:

- Upgrade react DOM for better errors and security

image
image
image

### CODE REVIEW

**Comment by simaob** [comment]
gonna check the functionality here. =)

**Comment by aabdaab** [comment]
Let's use `upperFirst` from lodash for the indicator names:
image

File: app/javascript/app/pages/country-context/socioeconomic-indicators/shared/bar-chart-tooltip/bar-chart-tooltip-component.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import styles from './bar-chart-tooltip-styles.scss';
+
+class CustomTooltip extends PureComponent {
+  render() {
+    const { content, config } = this.props;
+    const hasContent = content && content.payload && content.payload.length > 0;
+    const payload = content &&
+      content.payload &&
+      content.payload[0] &&
+      content.payload[0].payload;
```
Comment (aabdaab) [inline app/javascript/app/pages/country-context/socioeconomic-indicators/shared/bar-chart-tooltip/bar-chart-tooltip-component.jsx]: Tomek recently pointed out a cool function from lodash --> we could make it more readable by using `get` like so: `get(content, 'payload[0].payload')`

**Comment by Bluesmile82** [comment]
Ready again! Thanks!

**Comment by simaob** [comment]
but now with conflicts 😭

**Pull request merged**